docs/_source/guides/llms/practical_guides/set_up_annotation_team.md

```diff
@@ -92,14 +93,31 @@ for username, records in assignments.items():
     # create a dataset with their assingment and push to their workspace
     ds = rg.FeedbackDataset(fields=fields, questions=questions, guidelines=guidelines)
     ds.add_records(records)
-    ds.push_to_argilla(name="my_dataset", workspace=workspace.name)
 ```
 
+Then push the `FeedbackDataset` with the records to Argilla:
+
+::::{tab-set}
+
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+remote_dataset = dataset.push_to_argilla(name="my_dataset", workspace=workspace.name)
```

Comment:
Same here. Do we really need to save the `remote_dataset` if it's not going to be used later on? 
Also, I think it doesn't make sense to take this line of code out of the for loop above. If the original code snippet works in both versions then I would keep it as it is